src/official.md

```diff
@@ -27,6 +27,7 @@ Working groups:
 * [The Rust Language Design Team](https://lang-team.rust-lang.org/design_notes.html) - information related to the work of the lang team
 * [The Types Team](https://rust-lang.github.io/types-team/) - traits implementation improvements
 * [wg-async](https://rust-lang.github.io/wg-async/) - foundations of async I/O
+* [Const Generics Project Group](https://rust-lang.github.io/project-const-generics/index.html) - implements and designs the const_generics feature
```

Comment:
Please could you move this to the top of that group so the book names are in alphabetical order?